### ISSUE TITLE
Collections are not returned correctly

### ISSUE THREAD

**Issue by christophfriedrich** [Open-EO/openeo-r-backend#19]
I'm pretty sure that I managed to register at least one dataset correctly (see #18 about how I did that), but the response from `/collections/` is:

    {"collections":[{"name":{},"title":{},"description":{},"license":{},"extent":{},"links":{}}],"links":{"rel":"self","href":"http://localhost:8000/api/collections"}}

I.e. there is an item in the array (which is positive, because before I manually registered the data the array was completely empty), but it hasn't got any useful information...

**Comment by flahn** [comment]
Can you check if there are the following two file in each of the dataset folders:
`lookup.csv` and `md.json`. That was a change that I made recently. The md.json contains the STAC description except some values that are read dynamicly from the data.

Apart from that you need to always run loadDemo() (or even better the whole script I provided in the README). LoadDemo does as it says: it loads the demo processes and data into the server.

**Comment by christophfriedrich** [comment]
Yes, the files are both there, in both of the two folders, and they don't look corrupted either.

I always ran `loadDemo()` before I started the server (although I thought that it's only necessary when I want to DOWNload the demo data on the first launch). I tried again: Cleared the entire workspace (using the broom/brush icon in RStudio), executed the startup script and started the server. `/collections/` and `/processes/` are still empty (except for the JSON skeleton of course).

Loading the library gave a lot of output, but no errors. `createUsers` told me it's "Skipping creation. User already exists." The only error I encountered was in `loadDemo`:
> Fehler in CRS(x) : no system list, errno: 2

May this be the problem?
